### ISSUE TITLE
How to generate event files (npz files)?

### ISSUE THREAD

**Issue by jawonseojw** [lh9171338/FE-LSD#4]
I'd like to test some images with pretrained model.
However, it seems like that npz files are prepared in advance.
Please let me know how to generate those files from test images

**Comment by lh9171338** [comment]
Event data is captured by event cameras, e.g. DAVIS

**Comment by jawonseojw** [comment]
So you mean that this code can only be tested for images from event camera with event data?

**Comment by lh9171338** [comment]
Yes, the network requires two inputs, motion-blurred image and event data
![image](https://github.com/user-attachments/assets/f8b3b29f-3b0e-47dd-985e-088c8d06d1e6)
